wlog/svclog/svc1log/context.go

```diff
@@ -84,7 +84,7 @@ func safeAndUnsafeParamsFromParams(params []Param) (safe map[string]interface{},
 }
 
 // loggerFromContext returns the logger stored in the provided context. If no logger is set on the context, returns the
-// logger created by calling DefaultLogger.
+// logger created by calling defaultLogger.
```

Comment:
nit: should be `defaultLoggerCreator`.

Searching through the codebase, it looks like this typo/incorrect attribution happens for most of the logger types. Given that this PR doesn't modify this function, I would prefer that this not be corrected in this PR and we open a separate PR that fixes this documentation issue for all the relevant functions.